client/post-editor/editor-ground-control/index.jsx

```diff
@@ -205,6 +205,64 @@ export class EditorGroundControl extends PureComponent {
 		}
 	};
 
+	renderGroundControlQuickSaveButtons() {
+		const {
+			isSaving,
+			isSidebarOpened,
+			nestedSidebar,
+			post,
+			selectRevision,
+			setNestedSidebar,
+			toggleSidebar,
+			translate,
+		} = this.props;
+
+		const isSaveAvailable = this.isSaveAvailable();
+		const showingStatusLabel = this.shouldShowStatusLabel();
+		const showingSaveStatus = isSaveAvailable || showingStatusLabel;
+		const hasRevisions = isEnabled( 'post-editor/revisions' ) && get( post, 'revisions.length' );
+
+		if ( ! ( showingSaveStatus || hasRevisions ) ) {
+			return;
+		}
+
+		return (
+			<div className="editor-ground-control__quick-save">
```

Comment:
I struggled with the naming for this element. This is the best option I came up with, but am definitely open to improvement!

Reply:
That's a tough one, since they're also action buttons themselves (as with the action buttons on the right).
It'd be worth coming up with something really generic though - since the contents of this area could change to something that doesn't fit this description.
Would something like `primary-actions` / `secondary-actions` or `actions-left` / `actions-right` make sense? It'd involve renaming the original action buttons component but it could be worth while.